automatic/freecad/update_helper.ps1

```diff
@@ -32,9 +32,8 @@ param(
     }
     'portable' {
       $download_page = (Get-GitHubRelease -Owner "Freecad" -Name "Freecad" -Verbose).assets
-      $mobile = "portable"
```

Comment:
As per https://github.com/chocolatey-community/chocolatey-packages/pull/2403#discussion_r1498488093
@RedBaron2 suggested that the `$mobile` variables be kept in use.